cassandra-mesos-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/executor/jmx/NodeBackupJob.java

```diff
@@ -72,11 +77,13 @@ public void run() {
                     LOGGER.info("Starting backup on keyspace {}", keyspace);
                     keyspaceStarted();
 
-                    checkNotNull(jmxConnect).getStorageServiceProxy().takeSnapshot(backupName, keyspace);
-                    keyspaceFinished("SUCCESS", keyspace);
+                    final BackupManager backupManager = new BackupManager(checkNotNull(jmxConnect), backupDir);
+                    backupManager.backup(keyspace);
+
```

Comment:
As I've understood it is a generic coding style applied here, to use final for each variable, that is not modified in the code.